src/app/features/create-event/event-details-form/event-details-form.component.html

```diff
@@ -0,0 +1,138 @@
+<form class="details-form main-container" [formGroup]="eventDetailsForm" (ngSubmit)="handleSubmit()">
+  <div class="blue-container">
+    <h2 class="details-form__title font-heading-2">Dodaj wydarzenie {{ title }}</h2>
+    <mat-form-field color="primary">
+      <mat-label>Nazwa wydarzenia</mat-label>
+      <input matInput formControlName="name" />
+      <mat-error *ngIf="nameCtrl.errors as errors">
+        <ng-container *ngIf="errors['required']">To pole jest wymagane.</ng-container>
+        <ng-container *ngIf="errors['minlength'] as minLength"
+          >Min. liczba znaków: {{ minLength.actualLength }}/{{ minLength.requiredLength }}.</ng-container
+        >
+        <ng-container *ngIf="errors['maxlength'] as maxLength"
+          >Maks. liczba znaków: {{ maxLength.actualLength }}/{{ maxLength.requiredLength }}.</ng-container
+        >
+      </mat-error>
+    </mat-form-field>
+  </div>
+  <div class="toggle">
+    <mat-label class="toggle__label">Potwierdzenie obowiązkowe</mat-label>
+    <mat-slide-toggle color="primary" formControlName="isConfirmationRequired"></mat-slide-toggle>
+  </div>
+  <div class="toggle" *ngIf="isConfirmationRequiredCtrl.value">
+    <mat-label class="toggle__label">Ograniczona liczba miejsc</mat-label>
+    <mat-slide-toggle color="primary" formControlName="isLimitedPlaces"></mat-slide-toggle>
+  </div>
+  <fieldset class="fieldset">
+    <mat-form-field *ngIf="isLimitedPlacesCtrl.value" color="primary">
+      <mat-label>Liczba osób</mat-label>
+      <input matInput formControlName="limitedPlaces" type="number" />
+      <mat-error *ngIf="limitedPlacesCtrl.errors as errors">
+        <ng-container *ngIf="errors['required']">To pole jest wymagane.</ng-container>
+        <ng-container *ngIf="errors['min']">L. osób musi być większa od 0.</ng-container>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field color="primary" class="fieldset__input">
+      <mat-label>Data</mat-label>
+      <input matInput [matDatepicker]="picker" [min]="today" formControlName="startDate" />
+      <mat-error *ngIf="startDateCtrl.errors as errors">
+        <ng-container *ngIf="errors['required']">To pole jest wymagane.</ng-container>
+        <ng-container *ngIf="errors['matDatepickerMin']">Data nie może być w przeszłości.</ng-container>
```

Comment:
zamiast dodawać takie errory i pozwolić użytkownikowi na wybór tych dat, powinniśmy je zablokować by nie mógł ich wybierać

Reply:
W materialowym kalendarzu jest to już teoretycznie zablokowane i w większości sytuacji faktycznie nie da się wybrać, ale jak wpisywałem ręcznie parę razy to mi wchodziła data z przeszłości. Dlatego dodałem tak dodatkowo tego errora w razie czego